lucien-ios/Add New Comic Book/AddComicViewController.swift

```diff
@@ -0,0 +1,208 @@
+                         "Science Fiction",
+                         "Slice of Life",
+                         "Superhero",
+                         "Supernatural/Occult",
+                         "Suspense",
+                         "Western"
+    ]
+
+    // MARK: - Private IBOutlets
+
+    @IBOutlet private weak var coverPhotoLabel: UILabel!
+    @IBOutlet private weak var coverPhotoButton: UIButton!
+    @IBOutlet private weak var comicTitleLabel: UILabel!
+    @IBOutlet private weak var comicTitleTextField: UITextField!
+    @IBOutlet private weak var storyTitleLabel: UILabel!
+    @IBOutlet private weak var storyTitleTextField: UITextField!
+    @IBOutlet private weak var volumeLabel: UILabel!
+    @IBOutlet private weak var volumeTextField: UITextField!
+    @IBOutlet private weak var issueLabel: UILabel!
+    @IBOutlet private weak var issueTextField: UITextField!
+    @IBOutlet private weak var publisherLabel: UILabel!
+    @IBOutlet private weak var publisherTextField: UITextField!
+    @IBOutlet private weak var releaseDateLabel: UILabel!
+    @IBOutlet private weak var genreLabel: UILabel!
+    @IBOutlet private weak var conditionLabel: UILabel!
+    @IBOutlet private weak var selectADateButton: UIButton!
+    @IBOutlet private weak var selectAGenreButton: UIButton!
+    @IBOutlet private weak var selectAConditionButton: UIButton!
+    @IBOutlet private weak var datePicker: UIDatePicker!
+    @IBOutlet private weak var conditionPicker: UIPickerView!
+    @IBOutlet private weak var genrePicker: UIPickerView!
+    @IBOutlet private weak var scrollView: UIScrollView!
+    @IBOutlet private weak var releaseDateBottomLine: UIView!
+    @IBOutlet private weak var genreBottomLine: UIView!
+    @IBOutlet private weak var conditionBottomLine: UIView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureViewController()
+    }
+
+    // MARK: - Private Instance Methods
+
+    private func configureViewController() {
+        // Cover Photo Cell
+        coverPhotoLabel.font = UIFont(name: LucienTheme.Fonts.Muli.light.rawValue, size: 12)
+        coverPhotoLabel.textColor = LucienTheme.coolGrey
+
+        coverPhotoButton.titleLabel?.lineBreakMode = .byWordWrapping
+        coverPhotoButton.titleLabel?.textAlignment = .center
+        coverPhotoButton.setTitle("Take \n Cover \n Photo", for: .normal)
+        coverPhotoButton.titleLabel?.font = UIFont(name: LucienTheme.Fonts.Muli.bold.rawValue, size: 13)
+        coverPhotoButton.setTitleColor(LucienTheme.dark, for: .normal)
+        coverPhotoButton.titleEdgeInsets.top = 25
+        coverPhotoButton.layer.cornerRadius = 6
+        coverPhotoButton.clipsToBounds = true
+        coverPhotoButton.layer.borderWidth = 0.3
+        coverPhotoButton.layer.borderColor = UIColor.gray.cgColor
+
+        // Comic Title
+        cconfigureLabelwith(label: comicTitleLabel, fontName: LucienTheme.Fonts.Muli.light.rawValue, size: 12, color: LucienTheme.coolGrey)
```

Comment:
Really good thought with this helper function, but let's refactor it a little. Also watch out for typos